cmd/process-agent/api/status.go

```diff
@@ -9,21 +9,27 @@ import (
 	"encoding/json"
 	"net/http"
 
-	"github.com/DataDog/datadog-agent/pkg/status"
+	"github.com/DataDog/datadog-agent/pkg/process/util"
 	"github.com/DataDog/datadog-agent/pkg/util/log"
 )
 
 func statusHandler(w http.ResponseWriter, _ *http.Request) {
-	log.Trace("Received status request from process-agent")
+	log.Info("Got a request for the status. Making status.")
 
-	agentStatus, err := status.GetStatus()
+	agentStatus, err := util.GetStatus()
 	if err != nil {
-		_ = log.Warn("failed to get status from agent:", agentStatus)
+		if err != nil {
+			_ = log.Warn("failed to get status from agent:", agentStatus)
+			body, _ := json.Marshal(map[string]string{"error": err.Error()})
+			http.Error(w, string(body), http.StatusInternalServerError)
```

Comment:
nit: repeated with another handler below, perhaps a helper could help?

Reply:
That's a great idea! I've updated the PR with it!